modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -4893,7 +4893,7 @@ for(int i = 0; i < it2.count; i++, ++it2)
 }
 @endcode
 */
-class CV_EXPORTS LineIterator
+class CV_EXPORTS_W LineIterator
```

Comment:
Why do you need `create` function. Have you tried to use `CV_EXPORTS_W_SIMPLE` and add `CV_WRAP` to the  appropriate constructor?

Reply:
I tried it does not work (https://docs.opencv.org/4.x/da/d49/tutorial_py_bindings_basics.html) : 
_For large classes also, CV_EXPORTS_W is used. To extend class methods, CV_WRAP is used. Similarly, CV_PROP is used for class fields._ 

_Small classes/structs are extended using CV_EXPORTS_W_SIMPLE. These structs are passed by value to C++ functions. Examples are KeyPoint, Match etc. Their methods are extended by CV_WRAP and fields are extended by CV_PROP_RW._ 


I ran my code with CV_EXPORTS_W_SIMPLE and error 
  File "C:\lib\opencv\samples\python\lineiterator.py", line 8, in <module>
    pt = h.getPos()
AttributeError: 'cv2.LineIterator' object has no attribute 'getPos'